src/pages/comparison/components/comparison-element/comparison-element.component.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Profile } from './profile.component';
-import { Rate } from './rate.component';
-import { ROUTES } from '../../../shared/constants/routes.constants';
+import { Profile } from '../profile/profile.component';
+import { Rate } from '../rate/rate.component';
+import { ROUTES } from '../../../../shared/constants/routes.constants';
 
-import '../comparison.scss';
+import './comparison-element.scss';
```

Comment:
Doesn't look like CSS modules. Will you use it?

Reply:
Yes, i remember this.
I will add CSS modules in next pull request